samples/highcharts/demo/column-rotated-labels/demo.js

```diff
@@ -34,26 +34,26 @@ Highcharts.chart('container', {
     series: [{
```

Comment:
Shouldn't we change the chart.title too? It says 2014.

Reply:
Done in https://github.com/highcharts/highcharts/pull/7951/commits/3423c582c3f0290ac1131d028384906ea4827ede